src/main/flight/pid.c

```diff
@@ -1048,6 +1057,12 @@ void FAST_CODE pidController(const pidProfile_t *pidProfile, const rollAndPitchT
 
     const float tpaFactor = getThrottlePIDAttenuation();
 
+#ifdef USE_TPA_MODE
+    const float tpaFactorKp = (tpaMode == TPA_MODE_PD) ? tpaFactor : 1.0f;
```

Comment:
Another possibility would be to implement tpaKpRatio, to scale tpaFactor

```
const float tpaFactorKp = 1 - (1 - tpaFactor) *  tpaKpRatio;

Reply:
@ledvinap The current course of investigation is whether TPA on only D is better (and by implication TPA on P is not needed).  At this time we don't have any information that implies that the TPA effect on P should be some ratio of the effect on D. If we were going down that path it would probably make more sense to move to separate P/D configuration allow each to be configured with a different factor and break point.  But we're not sure that level of complication is needed. So this is an incremental step.